client/app/scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -1,16 +1,38 @@
 <div id="subnav" class="subnav" ng-class="{'auth-sendto': views.send}">
+
+    <div class="dropdown dropright lock-avatar pull-left" ng-if="item._locked" dropdown>
+        <figure class="avatar small avatar-with-info" dropdown-toggle>
+            <div class="logged-info"></div>
+            <div class="user-image">
+                <img sd-user-avatar data-src="item.lock_user.picture_url" data-initials="item.lock_user | username">
+            </div>
+        </figure>
+
+        <div class="dropdown-menu">
+            <div class="menu-label" translate>Locked by</div>
+            <figure class="avatar large">
+                <div class="user-image">
+                    <img sd-user-avatar data-src="item.lock_user.picture_url" data-initials="item.lock_user | username">
```

Comment:
By W3, img tag doesn't need to be closed at the end: http://www.w3schools.com/tags/tag_img.asp